.github/workflows/v8upgrade.yml

```diff
@@ -0,0 +1,48 @@
+name: V8 Upgrade
+
+on:
+  schedule:
+    - cron: '0 0 1 * *' # Run every 1st day of the month
+
+jobs:
+    upgrade:
+        name: Upgrade V8
+        runs-on: ubuntu-18.04
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+                  fetch-depth: 0
+            - name: Update depot_tools fetch config
+              run: cd deps/depot_tools && git config --unset-all remote.origin.fetch; git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
+              shell: bash
+            - name: Add depot_tools to PATH
+              run: echo "$PWD/deps/depot_tools" >> $GITHUB_PATH
+              shell: bash
+            - name: Fetch V8
+              run: cd deps && fetch v8 || true
+            - name: Get latest stable commit ref of v8
+              id: v8_latest_commit
+              run: echo ::set-output name=VALUE::$(curl -sS 'https://omahaproxy.appspot.com/all.json?os=linux&channel=stable' | jq '.[0]["versions"][0]["v8_commit"]')
+            - name: Get latest stable version number of v8
+              id: v8_latest_version
+              run: echo ::set-output name=VALUE::$(curl -sS 'https://omahaproxy.appspot.com/all.json?os=linux&channel=stable' | jq '.[0]["versions"][0]["v8_version"]')
+            - name: Checkout v8 to stable commit ref
+              run: cd deps/v8 && git fetch && git checkout ${{steps.v8_latest_commit.outputs.VALUE}}
+            - name: Copy deps/v8/include into deps/include
+              run: cp -r deps/v8/include/* deps/include/
```

Comment:
Will this remove any files that have been deleted?

Another step that we also need to take is to make sure we have the empty Go files that are imported so that the modules vendor folder works (there is a current problem where this does not work for the `internal` folder). This is a workaround to make sure the header files are copied to the vendor folder when using `go mod vendor`, otherwise folders without Go "code" gets ignored.

This is most tricky if any new folders get added

Reply:
@rogchap Thanks for the feedback. It is true that this step is the most tricky one. I'm going to look for ways to double-check all the requirements.

I'm thinking that to check ✅  everything we might as well have a script that would do everything for us and we can invoke it from the Github action, either way, I'm going to work on it and will ping once I have something that works for all cases: files being deleted, and new folders being added which would need to a `vendor.go` file and the extra import in `cgo.go`

Reply:
@rogchap I create a new Python script to take care of updating the include folder https://github.com/rogchap/v8go/pull/181/commits/945bd172813b996593c11a295814df774f05eab8

I have tested the script both without new folders and with new folders and looks like everything works as expected
Is my first writing Python so I'm not sure about best practices or anything so please feel free to point out my various Python flaws or improvements that can be applied.

The logic to update the `cgo.go` file assumes that the import for `_ "rogchap.com/v8go/deps/include"` will never change. From that assumption, I generate the list of imports to add from the directories inside the `deps/include` folder. Is this assumption correct?